examples/rbac/clusterrole/readonly.yaml

```diff
@@ -0,0 +1,10 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: readonly
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
```

Comment:
this gives read access to secrets, which makes the role escalating. I'd recommend the `view` role defined in-tree instead